doc/classes/Particles2D.xml

```diff
@@ -75,7 +79,15 @@
 			Number of vertical frames in [code]texture[/code].
 		</member>
 		<member name="visibility_rect" type="Rect2" setter="set_visibility_rect" getter="get_visibility_rect">
-			Editor visibility helper.
+			Visibility Helper
+			The [code]W[/code] and [code]H[/code] values control width and height of the visibility rectangle.
+			The [code]X[/code] and [code]Y[/code] values control the position of the upper-left corner of the visibility rectangle relative to the particle emitter.
+			Example: To match a box particle shape of [code]x:100 y:200[/code]
+			visibility_rect would be [code]w:100 h:200 X:-50 Y:-100[/code]
+			If the [code]visibility_rect[/code] is not in view the particles will not be rendered.
+			Tips: [code]visibility_rect[/code] can be automatically generated using the UI:
+			[Particles > Generate Visibility Rect]
+			Which appears on the toolbar when a [Particles2D] node is selected.
```

Comment:
This... feels very weird to read. I don't think all the extra fluff is needed -- just say `The rect used for visibility` or something of that nature. You don't need to explain Rect2 when we provide a link to Rect2's documentation.

Reply:
Okay? But we don't do this anywhere else we have a Rect2. Why would we need to do it here? Plus Rect2s kind of explain themselves...